ext/posix/tests/posix_ttyname_value_errors.phpt

```diff
@@ -19,11 +19,11 @@ foreach ($values as $value) {
     var_dump(posix_strerror(posix_get_last_error()));
 }
 ?>
---EXPECTF--
-Warning: posix_ttyname(): Argument #1 ($file_descriptor) must be between 0 and %d in %s on line %d
-bool(false)
-string(19) "Bad file descriptor"
+--EXPECTREGEX--
+Warning: posix_ttyname\(\): Argument #1 \(\$file_descriptor\) must be between 0 and \d+ in .+ on line \d+
+bool\(false\)
+string\(\d+\) "Bad file (descriptor|number)"
```

Comment:
why don't you just use EXPECTF with `%r(descriptor|number)%r` ? That's what you could use in other tests as well